ios/SecuritySdk/Classes/ASN.1/ASN1Type.swift

```diff
@@ -41,50 +41,65 @@ public enum Tag {
     
     /// ASN.1 types
     var type: Type {
-        switch self {
-        case let .boolean(type),
-            let .integer(type),
-            let .bitString(type),
-            let .octetString(type),
-            let .null(type),
-            let .objectIdentifier(type),
-            let .utf8String(type),
-            let .sequence(type),
-            let .set(type),
-            let .printableString(type),
-            let .ia5String(type),
-            let .utcTime(type):
-            return type
+        get {
```

Comment:
What does the get do?  I couldnt figure it out.

Reply:
since this is a var rather than a let due to it being a computed property, I rather limit its scope to just a getter so it doesnt get accidentally mutated.